python/ray/_private/utils.py

```diff
@@ -2026,6 +2026,9 @@ def pasre_pg_formatted_resources_to_original(
             continue
         result = PLACEMENT_GROUP_INDEXED_BUNDLED_RESOURCE_PATTERN.match(key)
         if result and len(result.groups()) == 3:
+            # This should be already skipped from the logic above.
```

Comment:
hmm that's right. Let me take a look at it again.

Reply:
ah interesting. it is because bundle_group_0 is technically a part of bundle_group_[group_id] because we don't have requirements for id length. 

I checked the bundle index before checking wildcard (alternatively, we can add id length restriction)

Reply:
addressed